README.md

```diff
@@ -1,19 +1,282 @@
+
+```js
+// File bathroom.js
+
+function sink() {
+  return 'Hi from the bathroom sink';
+}
+
+module.exports = {
+  sink: sink,
+}
+```
+
+And then we could call both `sink` functions in `app.js` and we'll never be unsure about which one we are calling.
+
+```js
+// File app.js
+
+var kitchen = require('./kitchen');
+var bathroom = require('./bathroom');
+
+console.log('I went to the kitchen sink and found', kitchen.sink());
+console.log('I went to the bathroom sink and found', bathroom.sink());
+```
+
+## Exporting More Than Just Functions
+
+So far we've only been exporting functions. What else can we export? The answer is anything! As long as we put it in a variable, we can export it. Take a look below for an example:
+
+```js
+// File closet.js
+
+var shirt = '👕';
+var pants = '👖';
+var shoes = {
+  white: '👟',
+  brown: '👞',
+}
+
+function workClothes() {
+  return shirt + ' ' + pants + ' ' + shoes.brown;
+}
+
+module.exports = {
+  shirt: shirt,
+  pants: pants,
+  shoes: shoes,
+  workClothes: workClothes,
+}
+```
+
+```js
+// File app.js
+
+var closet = require('./closet');
+
+console.log('Today I am going to wear', closet.workClothes(), 'to work');
+console.log('That means I am not going to wear', closet.shoes.white);
+```
+
+## What about code that isn't exported?
+
+We saw that we can pick which variables we want to export by including it in our export object (the object that we set `module.exports` equal to). What about variables that aren't in that export object?
+
+The short answer is that they will only be able to be used in the file they're in. In the example of `closet.js` above, if we didn't export `shoes` and our file looked like this:
+
+```js
+// File closet.js
+
+var shirt = '👕';
+var pants = '👖';
+var shoes = {
+  white: '👟',
+  brown: '👞',
+}
+
+function workClothes() {
+  return shirt + pants + shoes.brown;
+}
+
+module.exports = {
+  shirt: shirt,
+  pants: pants,
+  workClothes: workClothes,
+}
+```
+
+then `shoes` can only be accessed inside of `closet.js` like it is in the `workClothes` function.
+
+## Alternative Way To Export
+
+At the top of every Node file, you can imagine
+```js
+var exports = module.exports = {};
+```
+
+What this means for us is instead of setting `module.exports` equal to an object like we've been doing, we can instead assign properties. That means
+
+```js
+module.exports = {
+  shirt: shirt,
+  pants: pants,
+  workClothes: workClothes,
+}
+```
+
+```js
+module.exports.shirt = shirt;
+module.exports.pants = pants;
+module.exports.workClothes = workClothes;
+```
+
+```js
+exports.shirt = shirt;
+exports.pants = pants;
+exports.workClothes = workClothes;
+```
+
+Are all the same!
+
+Be careful of `exports` though. Make sure to only set properties on it, like `exports.shirt = shirt;`. If you ever write `exports = { shirt: shirt };`, it won't work.
```

Comment:
Could we give students an example of this to give them a handle on why it won't work? No need to go super in-depth.